Sources/XcodeGenKit/XCProjExtensions.swift

```diff
@@ -4,6 +4,59 @@ import XcodeProj
 
 extension PBXFileElement {
 
+    public struct NamePathPair: Comparable {
```

Comment:
Should this need to be public?

Move this struct into `PBXProjectGenerator` and you can change to `fileprivate`.